doc/index.html

```diff
@@ -372,9 +372,9 @@
         </h2>
       </div>
       <p>
-        The BLAST search algorithms don't directly understand FASTA files.
+        The BLAST search algorithms doesn't understand FASTA files.
```

Comment:
"do" goes with plural and "does" with singular, so the original was correct

Reply:
indeed, my fault. I will remove the change with a rebased re-commit